pkg/completions/completions_test.go

```diff
@@ -0,0 +1,108 @@
+package completions_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ionos-cloud/ionosctl/v6/pkg/completions"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/json2table"
+	"github.com/stretchr/testify/assert"
+)
+
+type innerStruct struct {
+	Str     *string  `json:"string,omitempty"`
+	Integer *int32   `json:"int32,omitempty"`
+	Float   *float64 `json:"float64,omitempty"`
+}
+
+type outerStruct struct {
+	InnerStructs *[]innerStruct `json:"innerStructs,omitempty"`
+}
+
+var (
+	innerStructJsonPaths = map[string]string{
+		"int32":   "int32",
+		"string":  "string",
+		"float64": "float64",
+	}
+
+	testStr     = "test-string"
+	testInt32   = int32(123)
+	testFloat64 = float64(12.5)
+
+	testInnerStruct = innerStruct{
+		Str:     &testStr,
```

Comment:
just a note that you can use `pointer.From("test-string")` to avoid declaring these variables before usage